bin/phpcbf

```diff
@@ -8,6 +8,6 @@
  * @license   https://github.com/squizlabs/PHP_CodeSniffer/blob/master/licence.txt BSD Licence
  */
 
-require_once __DIR__.'/../autoload.php';
+require_once __DIR__.'/../vendor/autoload.php';
```

Comment:
This (and other changes below) are BC break because this PATH would only exist, when PHP_CodeSniffer is located in `/vendor` folder, but not when it's installed in PEAR or as standalone package.